components/layout/AppHeader/AppHeader.jsx

```diff
@@ -1,20 +1,33 @@
-import { AppBar, makeStyles, Toolbar } from '@material-ui/core';
+import { AppBar, Box, makeStyles, Toolbar, Typography } from '@material-ui/core';
 import React from 'react';
 import Logo from '../../common/Logo/Logo';
+import { JoinUsHeaderButton } from '../JoinUsHeader/JoinUsHeaderButton';
 
-const useStyles = makeStyles(() => ({
-  root: {
-    backgroundColor: 'orangered',
+const useStyles = makeStyles(({ spacing }) => ({
+  menu: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    '& >*': {
+      padding: spacing(0, 1),
+    },
   },
 }));
 
 export const AppHeader = () => {
   const classes = useStyles();
 
   return (
-    <AppBar position="static" className={classes.root}>
+    <AppBar position="static" color="transparent" elevation={0}>
       <Toolbar>
         <Logo />
+        <Box m={1} className={classes.menu}>
+          <Typography>Menu 1</Typography>
+          <Typography>Menu 2</Typography>
+          <Typography>Menu 3</Typography>
+          <Typography>Menu 4</Typography>
+        </Box>
+        <JoinUsHeaderButton />
```

Comment:
JoinUs by mělo být součástí menu, pokud budeme dodržovat Mobil-first, aby se mohlo v hamburger menu schovat přímo do menu jako položka.

Reply:
@vojtabiberle v případě material UI nikoliv. Mobilní menu si musíme udělat sami. MAterial-ui na to přímo nic nemá. 

Takže musíme udělat jednou pro desktop a jednou pro mobil. Z figmy se navíc jeví, že celé menu bude dost custom. 

Každopádně jsem přidal nefunkční tlačítko pro mobil a skryl menu.